aptos-move/framework/supra-framework/doc/aptos_governance.md

```diff
@@ -1,7 +1,7 @@
 
-<a id="0x1_aptos_governance"></a>
+<a id="0x1_supra_governance"></a>
 
-# Module `0x1::aptos_governance`
+# Module `0x1::supra_governance`
```

Comment:
We should also rename the file.

Reply:
We can ignore this one, doc is auto-generated and it creates `supra_governance.md` automatically.